### DISCUSSION TITLE
New translation strings?

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#9593] [Firefly III] [answered]
Checking from time to time if there are new strings to translate and was suprised to see that many new ones. While the strings regarding subscriptions (new feature?) are valid, I am now seeing few pages of strings that I know I have already translated (multi-factor authentication). So why I must do thiem again?

**Comment by JC5** [comment, marked as answer]
I don't know :(. 

I noticed it myself too doing the Dutch strings. All my previous work is gone. I must have messed up the upload. I'm sorry about this. 

For what it's worth, Crowdin does suggest the previously translated entries so it's relatively easy to fill them in again, but it's not what I had expected.

**Reply by thehijacker** [reply]
Indeed suggestion works. But still lots of mouse clicks... No problem. I will finish Slovenian today. Have a wonderful new year 2025 and looking forward to new versions. Those subscriptions strings looks promising. I hope I could handle subscriptions like Spotify, Audible within Firefly III. I am now using Wallos (https://github.com/ellite/Wallos).

**Reply by JC5** [reply]
Ohhh I like that one! For the time being it is cosmetic only, but it opens new ideas yes 😊

**Reply by JC5** [reply]
And you of course, happy 2025 and thanks for translating!
